product-service/handlers/getProductById/getProductById.ts

```diff
@@ -0,0 +1,24 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import products from '../../mocks/products.json';
+import { getCORSHeaders } from '../../utils';
+import { errorMessages } from './errorMessages';
+import 'source-map-support/register';
+
+export const getProductById: APIGatewayProxyHandler = async (event, _context) => {
+  const { productId } = event.pathParameters;
```

Comment:
This line may fail on destructuring:
- you don't use try / catch
- event.pathParameters is not guaranteed to be an object:
```
export interface APIGatewayProxyEventBase<TAuthorizerContext> {
    ...
    pathParameters: { [name: string]: string } | null;
    ...
}
```